interfaces/HO-Portal/src/app/components/update-fsp/update-fsp.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { TranslateModule } from '@ngx-translate/core';
+import { UpdateFspComponent } from './update-fsp.component';
+
+describe('UpdateFspComponent', () => {
+  let component: UpdateFspComponent;
+  let fixture: ComponentFixture<UpdateFspComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [UpdateFspComponent],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      imports: [FormsModule, TranslateModule.forRoot()],
+    }).compileComponents();
```

Comment:
Above this line, the `providers`-array should be provided to reflect the contents of the `constructor`, from: https://github.com/global-121/121-platform/pull/2391/files#diff-8ba64fef935c2f040240510612e0a92f9626c6614317e660f5a4b54d6bec5fbdR45-R49

Reply:
So you shouldn't only try to include things the error-message asks for, but you should include what the **intent** of the component is.

It could be that the component requires a specific service only (and that service will 'under water' use the HttpClient), so for this test-scenario only that 1st level service is important, not every specific part down the 'dependency tree'.